plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/OpenShiftServerEditorSection.java

```diff
@@ -616,7 +594,7 @@ private Job loadingProjectsJob(OpenShiftServerEditorModel model, IServerWorkingC
 	private void loadResources(final Composite container, OpenShiftServerEditorModel model) {
 		IServerWorkingCopy server = input.getServer();
 
-		final Connection connection = OpenShiftServerUtils.getConnection(server);
+		OpenShiftServerUtils.getConnection(server);
```

Comment:
why is this operation even required if the result isnt used? If there's a reason for it we should add least comment it.